source/reference/api/reverse.txt

```diff
@@ -0,0 +1,90 @@
+.. _c2c-api-reverse:
+
+===========
+``reverse``
+===========
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Reverses the direction of a committed sync operation.
+
+For example:
+
+- You have a ``COMMITTED`` sync operation.
+
+- ``cluster0`` is the source and ``cluster1`` is the destination.
+
+- After the sync operation is ``COMMITTED``, new writes occur only on
+  the destination cluster.
+
+In this scenario, you can use the ``reverse`` endpoint to sync writes
+from ``cluster1`` to ``cluster0``.
+
+Requirements
+------------
+
+To use the ``reverse`` endpoint:
+
+- You must have started your original sync operation with the
+  ``reversible`` and ``enableUserWriteBlocking`` options set to
+  ``true``.
+
+- ``mongosync`` must be in the ``COMMITTED`` state.
+
+Request
+-------
+
+.. code-block:: http
+   :copyable: false
+
+   POST /api/v1/reverse 
+
+Request Body Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: includes/api/facts/no-body-parameters.rst
+
+Response
+--------
+
+.. include:: /includes/api/tables/basic-response.rst
+
+Example
+-------
+
+The following example reverses the direction of a committed sync operation.
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/reverse.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Behavior
+--------
+
+If the ``reverse`` request is successful, ``mongosync`` enters the
+``RUNNING`` state. Replication continues in the reverse direction from
+the original sync job.
```

Comment:
The **synchronization** continues in the reverse direction from the original sync job ........without the need to start from the beginning to copy the original data.